supertokens_python/supertokens.py

```diff
@@ -270,6 +293,7 @@ async def get_user_count(  # pylint: disable=no-self-use
         self,
         include_recipe_ids: Union[None, List[str]],
         tenant_id: Optional[str] = None,
+        user_context: Optional[Dict[str, Any]] = None,
```

Comment:
this is an internal function, do not set it to None by default. Same goes for other functions in this file

Reply:
These functions are called from supertokens_python/asyncio and supertokens_python/syncio